dashboard-backend/pkg/env/env.go

```diff
@@ -0,0 +1,18 @@
+package env
+
+import (
+	"os"
+
+	"github.com/joho/godotenv"
+	"github.com/litmuschaos/litmus-go/pkg/log"
+)
+
+func GoDotEnvVariable(key string) string {
+	// load .env file
+	err := godotenv.Load(".env")
+	if err != nil {
+		log.Fatalf("Error loading .env file")
```

Comment:
1. Do we need that comment?
2. log.Fatalf will do exit 1 and the further execution will stop so can we use `log.Errorf` instead?